cluster/gce/windows/configure.ps1

```diff
@@ -84,6 +84,20 @@ function FetchAndImport-ModuleFromMetadata {
   Import-Module -Force C:\$Filename
 }
 
+# Returns true if the ENABLE_NODE_LOGGING field in kube_env is true.
+# $KubeEnv is a hash table containing the kube-env metadata keys+values.
+function IsLoggingEnabled {
+  param (
+    [parameter(Mandatory=$true)] [hashtable]$KubeEnv
+  )
+
+  if ($KubeEnv.Contains('ENABLE_NODE_LOGGING') -and `
+      ($KubeEnv['ENABLE_NODE_LOGGING'] -eq 'true')) {
+    return $true
```

Comment:
I see that this currently looks like https://github.com/kubernetes/kubernetes/blob/master/cluster/gce/windows/common.psm1#L532, but can this just be?:

```
  return ($KubeEnv.Contains('ENABLE_NODE_LOGGING') -and `
      ($KubeEnv['ENABLE_NODE_LOGGING'] -eq 'true'))
```

Mostly just curious, if you don't want to bother trying it that's fine.